registrations/tasks.py

```diff
@@ -652,3 +653,50 @@ def run(self, user_id, **kwargs):
             raise
 
 pull_third_party_registrations = PullThirdPartyRegistrations()
+
+
+class SendPublicRegistrationNotifications(Task):
+    """
+    Send out a notification SMS to the CORP for public registrations not
+    converted to full registrations
+    """
+
+    def run(self):
+
+        messagesets = utils.search_messagesets(
+            {'short_name__contains': 'public'})
```

Comment:
I don't think that there is a filter that will filter these messagesets by short name contains. We'll probably have to do the filtering client side.